week1/matrix_done.py

```diff
@@ -0,0 +1,87 @@
+class Matrix:
+    def __init__(self, list):
+        self.rows= list
+    def print(self):
+        print('\n'.join(' '.join(map(str, sl)) for sl in self.rows))
+    def __eq__(self,other):
+        if self.rows==other.rows: return True
+        else: return False
+    def __add__(self, other):
+       m=Matrix([])
+       n=len(self.rows)
+       rows=[[0]*n for x in range(n)]
+       for i in range(0,n):
+          for j in range(0,n): rows[i][j]=self.rows[i][j]+other.rows[i][j]
+       m.rows=rows
+       return m
+    def __mul__(self, other):
+        ro=len(other.rows)
```

Comment:
Please use more descriptive variable names.
e.g.:
ro -> rows_other
co -> columns_other

is this what you meant?